src/Behat/Context/FileContext.php

```diff
@@ -46,4 +46,40 @@ public function createFileWithName($filename, $directory = NULL) {
     $this->files[] = $this->getCore()->createFileWithName($path, $directory);
   }
 
-}
\ No newline at end of file
+  /**
+   * Get path for file in drupal 8.
+   *
+   * @param string $filename
+   *   The name of the file to get.
+   * @param string directory
+   *   A string containing the files scheme, usually "public://".
+   *
+   * @throws Exception
+   *   Exception file not found.
+   *
+   * @throws Exception
+   *   Exception file could not be copied.
+   *
+   * @Given file get name :filename
+   * @Given file get name :filename in the :directory directory
+   */
+  public function visitFileWithName($filename, $directory = NULL) {
+    $public = 'public://';
+    $private = 'private://';
+
+    if (empty($directory) || $directory === $public) {
```

Comment:
What if we want to visit a file located in a subfolder?

For example: styles/my_style/image.png

Now we should write:
```
Given file get name 'my_style/image.png' in the 'public://' directory.
```
Could we make it works for 
```
Given file get name 'image.png' in the 'public://my_style' directory.
```
?

Same for private use case.

Reply:
You only defined private or public directory. The other path con from file definition,

Reply:
https://github.com/Metadrop/behat-contexts/blob/master/src/Behat/Cores/Traits/FileTrait.php#L20